docs/examples/index.rst

```diff
@@ -1,13 +0,0 @@
-Examples
```

Comment:
do we have this as a subsection anywhere now, or is this information gone altogether?

Reply:
Thank you, I meant to add this to the bottom of the "Using Modin" Section.